README.md

```diff
@@ -63,6 +94,10 @@ Currently, only FastCgiModule is supported.
 
 ## Versions
 
+### 1.7.0.0
```

Comment:
This should probably be changed to ### Unreleased per @vors [comment](https://github.com/PowerShell/xDSCResourceDesigner/pull/20#issuecomment-114978784)